Channels/Overview/02_Features.md

```diff
@@ -0,0 +1,30 @@
+# Key Features
+
+Manage sales channels easily with Actindo *Omni-Channel* orchestration:
+- Time-saving product information maintenance   
+  Products and product data are only created once and centrally in the *PIM*. This information is then displayed on the web store and all connected sales channels. There, you can decide individually for each channel which products and data should be displayed. You can easily connect new marketplaces at any time as well.
+
+- Link to inventory and order management   
+  Inventory levels are continuously synchronized between the marketplaces and the *Actindo Core1 Platform* and thus always kept up to date. Orders from the sales channels are automatically imported and passed on to the *Order management* module. This way, you always have an overview of all stocks and orders of your sales channels with just several clicks.
+
+- Data transformation via ETL   
+  In our *PIM*, you create your data in a media-neutral manner. It is then transformed via an automated ETL pipeline in the *DataHub* and made usable for every marketplace and connected system. Data that has been configured once can thus be used for every channel you need.
+
+- Tracking of orders   
+  Orders from the marketplaces are processed by the *Actindo Core1 Platform* in the same way as orders from your own online store. Order status and tracking codes are transmitted directly and automatically to your customers for tracking purposes.
+
+
+## Strengthen presence and reach
+
+**Easily integrate new touchpoints**
+
+The number of online marketplaces has more than doubled in recent years in the German-speaking countries alone. Online retailers have a multitude of potential sales channels at their disposal through which they can expand their reach and strengthen their customer presence.  
+
+ Take advantage of new opportunities and select the channels that can best benefit your business. The *Omni-Channel* module of the *Actindo Core1 Platform* allows you to connect as many marketplaces as you would like and continuously expand your sales channels. At the same time, the shopping experience of your customers is unified through a centralized control panel. This way, you benefit from new touchpoints and an up-to-date unified commerce environment.
+
+
+## Minimize effort and work efficiently
+
+**Increase sales &ndash; Conserve resources**
+
+Operating many different marketplaces can increase your sales, but at the same time it can be very time consuming. With the *Actindo Core1 Platform* and the integrated *Omni-Channel* module, you enrich your product data in the *PIM* only and then have it automatically distributed to all channels via ETL with the help of the *DataHub*. All product data maintenance and order management are being handled centrally via the *Actindo Core1 Platform*. This saves you and your employees valuable time and resources, that you can invest in other important processes for your company.  
```

Comment:
```suggestion
Operating many different marketplaces can increase your sales, but at the same time it can be very time consuming. With the *Actindo Core1 Platform* and the integrated *Omni-Channel* module, you enrich your product data in the *PIM* only and then have it automatically distributed to all channels via ETL with the help of the *DataHub*. All product data maintenance and order management are being handled centrally via the *Actindo Core1 Platform*. This saves you and your employees valuable time and resources that you can invest in other important processes for your company.  
```